password/src/Data/Password/Bcrypt.hs

```diff
@@ -183,6 +190,23 @@ checkPassword pass (PasswordHash passHash) =
       then PasswordCheckSuccess
       else PasswordCheckFail
 
+-- | Extract 'BcryptParams' from a 'PasswordHash' 'Bcrypt'.
```

Comment:
```hs
-- | Extracts the cost parameter as an 'Int' from a 'PasswordHash' 'Bcrypt'
```
instead of
```hs
-- | Extract 'BcryptParams' from a 'PasswordHash' 'Bcrypt'.
--
-- Returns 'Just BcryptParams' on success.
```
because there's no `BcryptParams`